src/com/maddyhome/idea/vim/KeyHandler.java

```diff
@@ -213,71 +213,60 @@ public void handleKey(@NotNull Editor editor,
     // If this is a "regular" character keystroke, get the character
     char chKey = key.getKeyChar() == KeyEvent.CHAR_UNDEFINED ? 0 : key.getKeyChar();
 
-    final boolean isRecording = editorState.isRecording();
-    boolean shouldRecord = true;
-
-    if (allowKeyMappings && handleKeyMapping(editor, key, context)) {
-      if (!editorState.isOperatorPending() || commandBuilder.peekCurrentCommandPartActualArgumentType() != Argument.Type.OFFSETS) {
-        return;
-      }
```

Comment:
I don't know what this is checking, and removing it didn't break any tests, which always makes me nervous. Perhaps there's a test scenario missing?

Reply:
Yes, I guess the tests for this scenario are in the IdeaVim-EasyMotion project. I'll check this code and add tests.